### ISSUE TITLE
Windows Pageant - No agent running

### ISSUE THREAD

**Issue by olirav** [keepassxreboot/keepassxc#5875]
## Overview
Unable to add SSH keys to a already running Pageant agent running on windows 10.
Was previously working on this machine, no obvious action that caused it to stop working.

## Steps to Reproduce
[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
1. Open database with keys in
2. Error message pops up
3. If attempting to add keys later via right click on the entry no response or action is taken.
4. "Add to agent" & "Remove from agent" buttons are greyed out in the keepass entries.

## Expected Behavior
Multiple SSH keys added to already running pegeant agent.

## Actual Behavior
Error message: "No agent running, cannot add identity."
No SSH keys added to the agent.

## Context
Previously worked - not aware of any deliberate changes that would break it.
Have tried updating to latest windows release of keepassxc (2.6.2) and Putty system 0.74
Have also tried a clean reinstall of Keepassxc

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - Version 2.6.2
Revision: e9b9582

Qt 5.15.1
Debugging mode is disabled.

Operating system: Windows 10 Version 1909
CPU architecture: x86_64
Kernel: winnt 10.0.18363

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (signed and unsigned sharing)
- YubiKey

Cryptographic libraries:
 libgcrypt 1.8.6

**Comment by droidmonkey** [comment]
Are you running pageant or keepassxc under a different user account?

**Comment by olirav** [comment]
No I have just manually run both from the desktop, and am not running either as admin.

**Comment by hifi** [comment]
KeePassXC detects running Pageant by looking up the window name of the Pageant instance. To check if the buttons are to be grayed out (and you have not enabled OpenSSH option in KeePassXC settings) it's using this snippet:

`return (FindWindowA("Pageant", "Pageant") != nullptr);`

So for some reason your KeePassXC instance cannot see the window of Pageant. The obvious reason has always been that one of them is being run under a different user account like said.

Please check KeePassXC settings and the SSH Agent page specifically if it has OpenSSH selected for some reason.

**Comment by olirav** [comment]
Ah looks like I had that option enabled, disabling it seems to have worked now.
I have been having this issue now and for a month or two and I remember that was one of the settings I tried changing when I tired to resolve it when it first popped up - but back then it made no difference on or off.
I am guessing there must have been some other issue with my set up that I fixed but I must have left that setting turned on.

Thank you for your help @hifi.
